src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -785,34 +790,78 @@ public void TestNestedDynamicAsVariable()
             Assert.AreEqual("Hello", _context["dynamic.lambda.name"]);
         }
 
+        private class ExpandoModel
+        {
+            public int IntProp { get; set; }
+            public string StrProp { get; set; }
+            public ExpandoObject Props { get; set; }
+        }
+
+        // Test case for [Issue #350](https://github.com/dotliquid/dotliquid/issues/350)
+        [Test]
+        public void TestNestedExpandoTemplate_Issue350()
+        {
+            var model = new ExpandoModel()
+            {
+                IntProp = 23,
+                StrProp = "from string prop",
+                Props = new ExpandoObject()
+            };
+            var dic = (IDictionary<string, object>)model.Props;

```

Comment:
Avoid uncommon abbreviations and use fully-form words from the English dictionary for code read-ability.

Reply:
Fixed, I blindly copied this test from issue #350 without cleaning it up.